pkg/cloud/aggregation.go

```diff
@@ -8,39 +8,64 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-var aggregationVarNames = []string{
-	"K6_CLOUD_AGGREGATION_MIN_SAMPLES",
-	"K6_CLOUD_AGGREGATION_PERIOD",
-	"K6_CLOUD_AGGREGATION_WAIT_PERIOD",
-	"K6_CLOUD_METRIC_PUSH_INTERVAL",
-	"K6_CLOUD_MAX_METRIC_SAMPLES_PER_PACKAGE",
-	"K6_CLOUD_MAX_METRIC_PUSH_CONCURRENCY",
+var aggregationVarNames = map[int][]string{
+	1: []string{
+		// cloud output v1: to be removed in the future
+		"K6_CLOUD_AGGREGATION_MIN_SAMPLES",
+		"K6_CLOUD_AGGREGATION_PERIOD",
+		"K6_CLOUD_AGGREGATION_WAIT_PERIOD",
+		"K6_CLOUD_METRIC_PUSH_INTERVAL",
+		"K6_CLOUD_MAX_METRIC_SAMPLES_PER_PACKAGE",
+		"K6_CLOUD_MAX_METRIC_PUSH_CONCURRENCY",
+	},
+	2: []string{
+		// cloud output v2
```

Comment:
@codebien if you have info that Cloud endpoints return additional parameters in case of cloud output v2 that should be set as env vars, please share. 

Related: https://github.com/grafana/k6-cloud/issues/1391

Reply:
If they are the env vars that will be overwritten then I'm fine with them.

Reply:
The question is more about what those env vars are supposed to be: I need to know which env vars to pass to k6 process.